0-bootstrap/README.md

```diff
@@ -62,13 +66,8 @@ Currently, the bucket information is replaced in the state backends as a part of
 
 ## Requirements
 
-If you are using `jenkins_bootstrap`, please see the [README](./modules/jenkins-agent/README.md) for the requirements.
-
-## Instructions
-
-If you are using `jenkins_bootstrap`, please follow the instructions on how to run the bootstrap step with the `jenkins_bootstrap` sub-module described in the [README](./modules/jenkins-agent/README.md), which include implementing VPN, configuring your Jenkins Master among other steps.
-
 ### Software
 
--   [gcloud sdk](https://cloud.google.com/sdk/install) >= 206.0.0
--   [Terraform](https://www.terraform.io/downloads.html) >= 0.12.6
+- [gcloud sdk](https://cloud.google.com/sdk/install) >= 206.0.0
+- [Terraform](https://www.terraform.io/downloads.html) = 0.12.24
```

Comment:
I am not sure if we should be enforcing `= 0.12.24`, we could keep it the same as before with the warning below if needed.

Reply:
changed via f68b382